src/main/java/gift/category/domain/Category.java

```diff
@@ -0,0 +1,54 @@
+package gift.category.domain;
+
+import gift.product.domain.Product;
+import gift.wish.domain.WishlistItem;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import java.util.List;
+
+@Entity
+@Table(name="categories")
+public class Category {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    Long id;
+    @NotNull
```

Comment:
애노테이션과 타입이 붙어있으면 보기가 어려워서 개행하는 것이 읽기 좋습니다ㅎㅎ
추가적으로 접근 제어자 private이 빠졌네요!
```suggestion
    
    @NotNull
```